src/StrawberryfieldUtilityService.php

```diff
@@ -28,6 +28,20 @@ class StrawberryfieldUtilityService implements StrawberryfieldUtilityServiceInte
 
   use StringTranslationTrait;
 
+  /**
+   * File system path pattern. Must begin and end with a word character, may have
+   * forward slashes delimiting directory names within the path.
+   */
+  const PATHPATTERN = "/(^(\w)+$)|(^(\w)([\/\w-]*)(\w)$)/";
```

Comment:
This is nice. Could we make this one more explicit? VALID_FILESYSTEM_PATH_PATTERN?

Reply:
Better now, I hope.